### DISCUSSION TITLE
Cannot execute examples/pinn_inverse/fractional_Poisson_2d_inverse.py

### DISCUSSION THREAD

**Discussion by AndresCenteno** [lululxvi/deepxde#1770] [Q&A]
Hello! I get this error message, literally got no clue on what to do, got python 3.8.10, paddle 2.6.1

Using backend: paddle
Other supported backends: tensorflow.compat.v1, tensorflow, pytorch, jax.
paddle supports more examples now and is recommended.
Traceback (most recent call last):
  File "trial.py", line 2, in <module>
    import deepxde as dde
  File "/home/aner/.local/lib/python3.8/site-packages/deepxde/__init__.py", line 20, in <module>
    from . import backend
  File "/home/aner/.local/lib/python3.8/site-packages/deepxde/backend/__init__.py", line 130, in <module>
    load_backend(get_preferred_backend())
  File "/home/aner/.local/lib/python3.8/site-packages/deepxde/backend/__init__.py", line 61, in load_backend
    mod = importlib.import_module(".%s" % mod_name.replace(".", "_"), __name__)
  File "/usr/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "/home/aner/.local/lib/python3.8/site-packages/deepxde/backend/paddle/__init__.py", line 1, in <module>
    from .tensor import *  # pylint: disable=redefined-builtin
  File "/home/aner/.local/lib/python3.8/site-packages/deepxde/backend/paddle/tensor.py", line 7, in <module>
    raise RuntimeError("DeepXDE requires PaddlePaddle==0.0.0(develop).")
RuntimeError: DeepXDE requires PaddlePaddle==0.0.0(develop).

Installed the 0.0.0 and got then

RuntimeError: (NotFound) The kernel `matmul_coo_dense` is not registered.   [Hint: Expected iter != kernels_.end(), but received iter == kernels_.end().] (at /paddle/paddle/phi/core/kernel_factory.cc:225)

**Comment by lululxvi** [comment]
@lijialin03 @HydrogenSulfate Any idea?

**Comment by HydrogenSulfate** [comment]
Can you provide more about the os version and environment information by: `python -c "import paddle; paddle.version.show()"` ? I have installed the develop version as below

![image](https://github.com/user-attachments/assets/2256c773-875a-430d-bab4-46800d7ab346)

and I can run `examples/pinn_inverse/fractional_Poisson_2d_inverse.py` sucessfully by running: `DDE_BACKEND=paddle python examples/pinn_inverse/fractional_Poisson_2d_inverse.py`

![image](https://github.com/user-attachments/assets/6057197d-dfe7-41e3-ba85-181e33343800)

you can check the installation script of paddle.

**Comment by HydrogenSulfate** [comment]
> RuntimeError: (NotFound) The kernel `matmul_coo_dense` is not registered. [Hint: Expected iter != kernels_.end(), but received iter == kernels_.end().] (at /paddle/paddle/phi/core/kernel_factory.cc:225)

Another reason is that `fractional_Poisson_2d_inverse.py` use sparse tensor API which requires CUDA>=11.2, 
`RuntimeError: (NotFound) The kernel matmul_coo_dense is not registered. [Hint: Expected iter != kernels_.end(), but received iter == kernels_.end().] (at /paddle/paddle/phi/core/kernel_factory.cc:225)` will be raised if you installed a paddle of cpu version. 

In summary, check if CUDA>=11.2 and install a corresponding gpu version paddle.
